docs/sphinx/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = source
+BUILDDIR      = build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
```

Comment:
I don't get dependency on "Makefile" and why Makefile is marked a .PHONY?

Reply:
Not sure, this was actually autogenerated by sphinx.

Reply:
Apologies, I used [sphinx-quickstart](https://www.sphinx-doc.org/en/master/man/sphinx-quickstart.html) (ships with sphinx itself) to autogenerate this file. I was focused on getting the whole system to work, didn't look at how ugly this Makefile was. Will improve.